pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/AggregateOperator.java

```diff
@@ -104,6 +113,7 @@ public AggregateOperator(Operator<TransferableBlock> inputOperator, DataSchema d
     _resultSchema = dataSchema;
     _readyToConstruct = false;
     _hasReturnedAggregateBlock = false;
+    _operatorStats = new OperatorStats(context.getRequestId(), context.getStageId(), EXPLAIN_NAME);
```

Comment:
in case we have subclasses going forward, let's call the explainString method b/c EXPLAIN_NAME can be overwritten
```suggestion
    _operatorStats = new OperatorStats(context.getRequestId(), context.getStageId(), toExplainString());
```

Reply:
Changed the code a little bit and we have to use EXPLAIN_NAME. PTAL.